extension/popup/popupView.jsx

```diff
@@ -863,6 +885,13 @@ const SearchResultsContent = ({
     onNextSearchResultClick();
   };
 
+  const onMusicServiceChange = event => {
+    if (event) {
```

Comment:
Is there any reason why the event would be false/null?

Reply:
I really do not think so cause before this is even called an event is being triggered